src/install/extract_tarball.zig

```diff
@@ -203,22 +203,7 @@ fn extract(this: *const ExtractTarball, tgz_bytes: []const u8) !Install.ExtractD
         }
 
         const Archive = @import("../libarchive/libarchive.zig").Archive;
-        const Zlib = @import("../zlib.zig");
-        var zlib_pool = Npm.Registry.BodyPool.get(default_allocator);
-        zlib_pool.data.reset();
-        defer Npm.Registry.BodyPool.release(zlib_pool);
 
-        var zlib_entry = try Zlib.ZlibReaderArrayList.init(tgz_bytes, &zlib_pool.data.list, default_allocator);
```

Comment:
I removed this because libarchive already handles the compression, and it allows me to add zstd to libarchive.zig.